packages/plugin-ext/src/plugin/workspace.ts

```diff
@@ -80,6 +84,7 @@ export class WorkspaceExtImpl implements WorkspaceExt {
     initialize(): void {
         this.proxy = this.rpc.getProxy(Ext.WORKSPACE_MAIN);
         this.logger = new PluginLogger(this.rpc, 'workspace');
+        this.encodingService = new EncodingService();
```

Comment:
Is this not injectable?

Reply:
I added some bindings for the EncodingService, and used the injection rather than creating  a new one.